sound/soc/sof/sof-priv.h

```diff
@@ -62,6 +62,12 @@
 
 #define DMA_CHAN_INVALID	0xFFFFFFFF
 
+/* DSP D0ix sub-state */
+enum sof_d0_substate {
+	SOF_DSP_D0I0 = 0,	/* DSP default D0 substate */
+	SOF_DSP_D0I3,		/* DSP D0i3(low power) substate*/
+};
+
 struct snd_sof_dev;
```

Comment:
@keyonjie do you have any document describing low power states of the DSP? I'm not very familiar with low power states in general, so I'll might ask some dummy questions on this PR :)

Reply:
@dbaluta there's some information on PM in this discussion.
https://github.com/orgs/thesofproject/teams/sof-developers/discussions/6

Reply:
@ranj063 thanks!

Reply:
> @keyonjie do you have any document describing low power states of the DSP? I'm not very familiar with low power states in general, so I'll might ask some dummy questions on this PR :)

Yes, as @ranj063 listed, we have discussion in the sof-developers. By the way @dbaluta, are there similar low power state in your Audio DSP?